hoprd/rest-api/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hoprd-api"
-version = "3.10.0"
+version = "3.10.1"
```

Comment:
It deserves likely a minor version increase, since you're adding stuff, not just fixing.